PCAxis.Web.Controls/VariableSelector/VariableSelectorValueSelectCodebehind.vb

```diff
@@ -585,7 +585,7 @@ Public Class VariableSelectorValueSelectCodebehind
         End Select
 
         ' --- MandatoryText
-        MandatoryText.Text = GetLocalizedString("CtrlVariableSelectorMandatoryText")
+        MandatoryText.Text = "(" + GetLocalizedString("CtrlVariableSelectorMandatoryText") + ")"
```

Comment:
It would be better if we only changed the text in the language file instead. That way everybody is not forced to have the text in parentheses

Reply:
Good point. I have now updated and moved the parantheses to the language file instead of in the codebehind.